### DISCUSSION TITLE
使用封面双栏英文标题所引起的行距过大的问题

### DISCUSSION THREAD

**Discussion by wang19950602** [TheNetAdmin/zjuthesis#192] [General]
您好，我在使用模板中zjuthesis.tex文件中的双栏中英文标题命令时，出现了英文标题两栏的行距过大，导致占用空间过多，相应的申请人姓名、指导教师等本该出现在与标题同一页的信息，被挤到了下一页中，请问这一问题应如何解决。

我使用zjuthesis.tex中双栏标题命令的方式如下所示。
![提问1](https://user-images.githubusercontent.com/32090473/132791694-719e3a78-3afe-4812-b9df-61e524d7cb90.png)
![提问2](https://user-images.githubusercontent.com/32090473/132792785-06d42a44-349c-4636-ba50-f0df5f680771.png)

**Comment by TheNetAdmin** [comment]
哦这似乎是因为同时用了中文和英文的双行造成的，之前没测过同时用的效果

这样吧，你提个issue填一下issue模板里的信息，我试一下能不能复现

**Reply by wang19950602** [reply]
OK，我填好了，您看看是否合适

**Reply by Jackie-zk** [reply]
您好，我也遇到了，这个应该如何解决呢

**Reply by TheNetAdmin** [reply]
#193
